src/styles/partials/components/_site-status.scss

```diff
@@ -0,0 +1,9 @@
+.status-text{
+    color: $gray-light;
+    font-size: $h2-font-size;
+    text-transform: uppercase;
+    font-family: $default-heading-font;
+    font-weight: $font-medium;
+    letter-spacing: $default-padding;
+    padding-left: 3rem;
```

Comment:
please use $padding-jumbo @sdurphy we can figure out what is needed as part of this pr.

Reply:
done